Dockerfile

```diff
@@ -21,7 +24,7 @@ COPY src/ /src/
 RUN python -m venv /venv && \
     /venv/bin/pip install --no-cache-dir -U pip nuitka setuptools wheel && \
     /venv/bin/pip install --no-cache-dir --no-warn-script-location -r ./requirements.txt && \
-    /venv/bin/python -m nuitka --onefile /src/harbor.py && \
+    /venv/bin/python -m nuitka --onefile --clean-cache=all /src/harbor.py && \
```

Comment:
Do we need to add the `--clean-cache=all`?

Reply:
No, seems like this is not needed.